inference_segmentation.py

```diff
@@ -1,12 +1,14 @@
+import os
 import csv
-from math import sqrt
 from tqdm import tqdm
 from pathlib import Path
 from numbers import Number
 from tempfile import mkstemp
 from omegaconf import DictConfig
 from typing import Dict, Sequence, Union
 from dataset.stacitem import SingleBandItemEO
+import rasterio
+from shutil import move
```

Comment:
can we start organizing module loading according to PEP8? i.e. system libraries in the first block, external in the second, locally written libraries in the third.

Reply:
Changed it in this script.  
Following the refactoring of all the other modules, this will be done in the other module as well.